### PR TITLE
3694 notification ui test

### PR DESCRIPTION
**Pull request by 6a68**
Still needs a tiny bit of cleanup, but want to get feedback on whether the test colors should match photon design style or not (currently matches invision, not photon).

### CODE REVIEW

File: static/css/partials/_theme.scss [hunk 1]
```diff
@@ -27,6 +27,75 @@
...
 }
 
+// placeholder for download a/b test of notification bar
+// TODO: use photon colors or existing site style colors instead?
+$test-teal: #00a0c0;
+$test-link-blue: #0297f8;
+$test-yellow: #ffe900; // photon design system: yellow 50
+$test-blue: #0099f6;
+$test-lt-grey: #efeff1;
+$test-dk-grey: #37383d;
```
Comment (6a68) [inline static/css/partials/_theme.scss]: @johngruen Are these colors ok to use? Used a color picker to pull them from the invision pics, but they don't match the photon design system colors.

File: server/src/pages/shot/view.js [hunk 2]
```diff
@@ -429,8 +429,13 @@ class Body extends React.Component {
   }
 
   renderFirefoxRequired() {
-    return <div className="highlight-color-scheme alt-notification">
-      <div>
+    let showTestStyle;
```
Comment (chenba) [inline server/src/pages/shot/view.js]: CI failure is from this unused var.

File: static/img/firefox-logo.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...7.92,427.35,321,452.18,321,452.18c-4.27,1-15.68,9.16-19.72,17.82-4.27,10.83-12.07,35.28,11.55,60.9,37.09,40.19-76.2,95.36,98.66,199.57,4.41,2.4-41-1.43-61.64-10.36Z" style="fill:url(#linear-gradient-4)"/><path d="M325.07,657.5c49.44,17.21,107,14.19,141.52-4.86,23.09-12.85,52.7-33.43,70.92-28.35-15.78-6.24-27.73-9.15-42.1-9.86-2.45,0-5.38,0-8-.32a136,136,0,0,0-15.76.86c-8.9.82-18.77,6.43-27.74,5.53-.48,0,8.7-3.77,8-3.61-4.75,1-9.92,1.21-15.37,1.88-3.47.39-6.45.82-9.89,1-103,8.73-190-55.81-190-55.81-7.41,25,33.17,74.3,88.52,93.57Z" style="fill:url(#linear-gradient-5)"/><path d="M813.74,801.65c104.16-102.27,156.86-226.58,134.58-366,0,0,8.9,71.5-24.85,144.63,16.21-71.39,18.1-160.11-25-252C841,205.64,746.45,141.11,710.35,114.19,655.66,73.4,633,31.87,632.57,23.3c-16.34,33.48-65.77,148.2-5.31,247,56.64,92.56,145.86,120,208.33,205C950.67,631.67,813.74,801.65,813.74,801.65Z" style="fill:url(#linear-gradient-6)"/><path d="M798.81,535.55C762.41,460.35,717,427.55,674,392c5,7,6.23,9.47,9,14,37.83,40.32,93.61,138.66,53.11,262.11C659.88,900.48,355,791.06,323,760.32,335.93,894.81,561,959.16,707.6,872,791,793,858.47,658.79,798.81,535.55Z" style="fill:url(#linear-gradient-7)"/></g></g></g></g></svg>
```
Comment (chenba) [inline static/img/firefox-logo.svg]: We should remove the `xmlns:xlink` (per #3063).

File: server/src/pages/shot/view.js [hunk 4]
```diff
@@ -429,8 +429,11 @@ class Body extends React.Component {
   }
 
   renderFirefoxRequired() {
-    return <div className="highlight-color-scheme alt-notification">
-      <div>
+    const testStyle = this.props.abTests.firefoxAd && this.props.abTests.firefoxAd.value || "";
+    return <div className={`${testStyle} highlight-color-scheme alt-notification`}>
+      <div style={{marginLeft: "auto", marginRight: "auto", fontSize: "14px"}}>
```
Comment (chenba) [inline server/src/pages/shot/view.js]: Is there a reason for the inline style?

File: static/img/firefox-logo.svg (hunk 3 above)
Comment (6a68) [reply static/img/firefox-logo.svg]: note from ian in irc: "I think ./node_modules/.bin/svgo will clean up anything in an svg file"

**Comment by 6a68** [comment]
@johngruen Before I rebase and land this, could you confirm that the custom dimension `cd8` is correct for reporting the A/B test results (#3899)?

**Comment by johngruen** [comment]
Checking...

**Comment by johngruen** [comment]
@6a68 set up!

image

**Comment by 6a68** [comment]
We have some other open issues around A/B testing, including whether we should use our own system or the GA-based system. I'm going to close this and track what to do next in #3694.

**Pull request closed without merging**